analytics_data_api/utils.py

```diff
@@ -77,7 +79,11 @@ def consolidate_answers(problem):
                 consolidated_answer.variant = None
                 consolidated_answer.consolidated_variant = True
             else:
```

Comment:
I tend to find if statements easier to read in the positive.  I know the `if not consolidated_answer:` statement (line 77) isn't in touched code, but would it be possible to switch the if/else here so it's `if consolidated_answer:`?